schemas/context/experienceevent.schema.json

```diff
@@ -44,7 +44,7 @@
           "title": "End User IDs",
           "$ref": "https://ns.adobe.com/xdm/context/enduserids",
           "description":
-            "Condensed, normalized encapsulation of all end user identifiers.\n"
+            "Condensed, normalized encapsulation of all end user identifiers."
```

Comment:
Looks like this is an formatting change which slipped into this PR. I think we should steer clear from such side-cars in the future. For now, I think we can let that pass.